anchore_engine/services/policy_engine/engine/vulns/mappers.py

```diff
@@ -900,4 +907,15 @@ def to_engine_vulnerabilities(self, grype_vulnerabilities):
                     }
                 )
 
+        for _, vulnerability in intermediate_tuple_list.items():
+            unique_affected_packages = set(
+                [
```

Comment:
(nit) is the list comprehension necessary here? set can operate over an iterable

Reply:
you are correct. i keep forgetting that i can just put the expression in there